freebsd/sys/sys/event.h

```diff
@@ -55,7 +56,7 @@
 	(kevp)->udata = (f);			\
 } while(0)
 
-struct kevent {
+    struct kevent {
```

Comment:
Why?

Reply:
I have clang-format plugin in vim and I forget to doable it.